src/pages/Script/ScriptsComp.tsx

```diff
@@ -122,33 +130,46 @@ export const ScriptCells = ({
 }) => {
   const history = useHistory()
   const { codeHash, hashType } = useParams<{ codeHash: string; hashType: string }>()
-
-  const cellsQuery = useQuery([`scripts_${cellType}`, codeHash, hashType, page, size], async () => {
-    const { data, meta } = await explorerService.api.requesterV2
-      .get(`scripts/${cellType}`, {
-        params: {
-          code_hash: codeHash,
-          hash_type: hashType,
-          page,
-          page_size: size,
-        },
-      })
-      .then((res: AxiosResponse) =>
-        toCamelcase<Response.Response<{ deployedCells?: CellInScript[]; referringCells?: CellInScript[] }>>(res.data),
-      )
-    const camelCellType = camelcase(cellType) as 'deployedCells' | 'referringCells'
-    if (data == null) {
-      throw new Error('Fetch Cells null')
-    }
-    const cells = data[camelCellType]!
-    if (cells == null || cells.length === 0) {
-      throw new Error('Cells empty')
-    }
-    return {
-      total: meta?.total ?? 0,
-      cells,
-    }
+  const [cellsEmpty, setCellsEmpty] = useState<Record<'deployedCells' | 'referringCells', boolean>>({
+    deployedCells: false,
+    referringCells: false,
   })
+
+  const camelCellType = camelcase(cellType) as 'deployedCells' | 'referringCells'
+
+  const cellsQuery = useQuery(
+    [`scripts_${cellType}`, codeHash, hashType, page, size],
+    async () => {
+      const { data, meta } = await explorerService.api.requesterV2
+        .get(`scripts/${cellType}`, {
+          params: {
+            code_hash: codeHash,
+            hash_type: hashType,
+            page,
+            page_size: size,
+          },
+        })
+        .then((res: AxiosResponse) =>
+          toCamelcase<Response.Response<{ deployedCells?: CellInScript[]; referringCells?: CellInScript[] }>>(res.data),
+        )
+      if (data == null) {
+        setCellsEmpty({ ...cellsEmpty, [camelCellType]: true })
+        throw new Error('Fetch Cells null')
+      }
+      const cells = data[camelCellType]!
+      if (cells == null || cells.length === 0) {
```

Comment:
`==` should be avoided and `===` should be used

Reply:
`cells.lenght === 0` should not thrown because it's a valid situation

Reply:
> `cells.lenght === 0` should not thrown because it's a valid situation

I think it is used to show this page from previous consideration.

![image](https://github.com/Magickbase/ckb-explorer-frontend/assets/5457564/9ae82503-ac62-45cb-b090-46b56db3605d)

Reply:
I would consider it's confusing because `empty set` is not `none` or `not found`

Reply:
> I would consider it's confusing because `empty set` is not `none` or `not found`

But I didn't find any other page for this case, so I just followed how it was done before.
Do you mean there would be another page designed?

Reply:
It can be simply marked with `no records` as https://explorer.nervos.org/nft-collections/0x3947f34552cf247cb3bce1c65b12be7ca069d6e5bb18a7d1421c1b8471bf520f

Reply:
> It can be simply marked with `no records` as https://explorer.nervos.org/nft-collections/0x3947f34552cf247cb3bce1c65b12be7ca069d6e5bb18a7d1421c1b8471bf520f

Sure, I will change this way~

Reply:
> It can be simply marked with `no records` as https://explorer.nervos.org/nft-collections/0x3947f34552cf247cb3bce1c65b12be7ca069d6e5bb18a7d1421c1b8471bf520f

Can we use the same UI in this page?

Reply:
I think it's fine to reuse it because it's relatively simple